server/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
```

Comment:
Move the app = express() and the port = 3002 below your other require statements.